parsec/core/fs/remote_loader.py

```diff
@@ -26,9 +37,28 @@
     FSBadEncryptionRevision,
     FSWorkspaceNoReadAccess,
     FSWorkspaceNoWriteAccess,
+    FSUserNotFoundError,
+    FSDeviceNotFoundError,
+    FSInvalidTrustchainEror,
 )
 
 
+@contextmanager
+def translate_remote_device_manager_errors():
+    try:
+        yield
+    except RemoteDevicesManagerError as exc:
```

Comment:
This should be last otherwise it will swallow all the exceptions types...
(I have the feeling we are missing some tests if such error went through 😄 )

Reply:
4f609d1